docs/how-to/debugging.rst

```diff
@@ -2,6 +2,8 @@
    :description: How to debug using HIP.
    :keywords: AMD, ROCm, HIP, debugging, ltrace, ROCgdb, WinGDB
 
+.. _debugging_with_hip:
+
```

Comment:
Comments for the intro:
![image](https://github.com/ROCm/HIP/assets/164940351/8cb7c60f-e58f-44ed-9c08-317661da85e0)

Should it start with:
AMD ROCm ...
ROCM HIP ...

AMD debugging is ambiguous. 

The highlighted items don't need italics. 
The yellow items should be hyperlinked to relevant docs.

Reply:
Fixed. Please check and push resolved, if it's acceptable.